deeprank_gnn/models/forcefield/top.py

```diff
@@ -1,7 +1,7 @@
 
 
 class TopRowObject:
-    def __init__(self, residue_name, atom_name, kwargs):
+    def __init__(self, residue_name: str, atom_name: str, kwargs: dict):
```

Comment:
shouldn't kwargs become Dict[str, str] ? Or is this the default?

Reply:
Good point. I changed it to `Dict[str, any]`